### PR TITLE
Bump the OCP Version on cloud-bulldozer/e2e-benchmarking

### PR DESCRIPTION
**Pull request by krishvoor**

### CODE REVIEW

File: ci-operator/config/cloud-bulldozer/e2e-benchmarking/cloud-bulldozer-e2e-benchmarking-master.yaml [hunk 1]
```diff
@@ -271,6 +274,7 @@ tests:
       COMPUTE_NODE_TYPE: m6g.xlarge
       CONTROL_PLANE_INSTANCE_TYPE: m6g.xlarge
       ENABLE_LAYER_3: "false"
+      FEATURE: NetworkSegmentation
```
Comment (josecastillolema) [inline ci-operator/config/cloud-bulldozer/e2e-benchmarking/cloud-bulldozer-e2e-benchmarking-master.yaml:277]: Is the feature needed on OCP 4.18?

Comment (krishvoor) [reply ci-operator/config/cloud-bulldozer/e2e-benchmarking/cloud-bulldozer-e2e-benchmarking-master.yaml:277]: Only for UDN builds, something similar to what we do here:
https://github.com/openshift/release/blob/master/ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.18-nightly-x86.yaml#L297-L305

**Review by krishvoor — COMMENTED** [review]
/pj-rehearse ack

**Pull request merged**